### ISSUE TITLE
About the epochs

### ISSUE THREAD

**Issue by nemonameless** [ultralytics/ultralytics#183]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

 All detection models are trained with 300 epochs?  just find 100epoch in default.yaml

**Comment by AyushExel** [comment]
@nemonameless yeah the default is set to 100 as they use pre-trained weights by default. It you're training from scratch, you can use more epochs

**Comment by nemonameless** [comment]
> pre-trained weights

Thanks, so what and where is the pre-trained weight?  trained on ImageNet just for backbone？

**Comment by Laughing-q** [comment]
@nemonameless all the detection models are trained with 500 epochs.

**Comment by nemonameless** [comment]
Is yolov8n-cls.pt the pre-training weight for detection backbone？

**Comment by AyushExel** [comment]
@nemonameless no detection models have been trained from scratch independently of cls models.. cls models have been trained for 90epochs

**Comment by nemonameless** [comment]
> @nemonameless all the detection models are trained with 500 epochs.

Where can we see this setting?

**Comment by AyushExel** [comment]
> > @nemonameless all the detection models are trained with 500 epochs.
> 
> Where can we see this setting?

We'll the commands to reproduce on the README soon

**Comment by glenn-jocher** [comment]
@AyushExel ah that's right, we're missing reproduce commands.

**Comment by nemonameless** [comment]
Is the update complete now? 
Can we run the command line to reproduce the training results of modelzoo now? Thanks
